src/Snowflaqe.fsproj

```diff
@@ -7,13 +7,10 @@
     <PackAsTool>true</PackAsTool>
     <IsPackable>true</IsPackable>
     <RollForward>Major</RollForward>
-    <TargetFramework>net5.0</TargetFramework>
-    <Version>1.24.0</Version>
-    <PackageReleaseNotes>
-      Fix copyLocalLockFileAssemblies option.
-      Update Fable.Remoting.Json and SimpleJson to latest.
-      Lower required FSharp.Core version to v4.7.2
-    </PackageReleaseNotes>
+    <TargetFrameworks>netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <Version>1.25.0</Version>
+    <PackageReleaseNotes>Added support for System.Text.Json</PackageReleaseNotes>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Not a big fan of BindingRedirects, why is this needed here? Please try to remove it from here